File: CS/WindowsApplication39/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraWizard;

namespace WindowsApplication39
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            ClearWizardPageCollection(wizardControl1.Pages);
            WizardPage newPage = new WizardPage();
            newPage.Text = "This is new wizard page";
            wizardControl1.Pages.Add(newPage);
        }

        private void ClearWizardPageCollection(WizardPageCollection wizardPageCollection)
        {
            for (int i = wizardPageCollection.Count  - 1; i >= 0; i--)
            {
                wizardPageCollection[i].Dispose(); 
            }
        }
    }
}

File: VB/WindowsApplication39/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraWizard

Namespace WindowsApplication39
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			ClearWizardPageCollection(wizardControl1.Pages)
			Dim newPage As New WizardPage()
			newPage.Text = "This is new wizard page"
			wizardControl1.Pages.Add(newPage)
		End Sub

		Private Sub ClearWizardPageCollection(ByVal wizardPageCollection As WizardPageCollection)
			For i As Integer = wizardPageCollection.Count - 1 To 0 Step -1
				wizardPageCollection(i).Dispose()
			Next i
		End Sub
	End Class
End Namespace